modules/siddhi-query-api/src/main/java/org/wso2/siddhi/query/api/definition/AggregationDefinition.java

```diff
@@ -88,7 +90,15 @@ public AggregationDefinition annotation(Annotation annotation) {
         return this;
     }
 
-    public List<Annotation> getAnnotation() {
+    public void setOutputAttributeExpressions(Expression expressions) {
```

Comment:
I added this, so that we can set the expressions required to get the final output from intermediate data. For example, avg = sum/count is exposed as an expression in AvgIncrementalAttributeAggregator. We need to make this information available in for the retrieval part to be done. I can remove this method and just pass the expressions as a separate list if necessary.